src/app/component/home/components/eco-events/eco-events.component.html

```diff
@@ -4,48 +4,17 @@
       <div class="main-event-image">
         <img [src]="latestNews[0]?.imagePath || eventImg" alt="event-image">
       </div>
```

Comment:
Why use extra and unnecessary elements? Why those covered constructions, like
```js
<div class='...'>
  <myelement>
  </myelement>
</div>
```
instant of simple constructions :
```
<myelement class=...>
</myement>
```
?

Reply:
Fixed